pkg/executor/show_stats.go

```diff
@@ -146,27 +146,24 @@ func (e *ShowExec) appendTableForStatsMeta(dbName, tblName, partitionName string
 	if statsTbl.Pseudo {
 		return
 	}
+	row := make([]any, 8)
```

Comment:
```go
	if statsTbl.Pseudo {
		return
	}
	lastAnalyzeTime := any(nil)
	if statsTbl.IsAnalyzed() {
		lastAnalyzeTime = e.versionToTime(statsTbl.LastAnalyzeVersion)
	}
	lastStatsFullUpdateTime := any(nil)
	if statsTbl.LastStatsFullUpdateVersion != 0 {
		lastStatsFullUpdateTime = e.versionToTime(statsTbl.LastStatsFullUpdateVersion)
	}
	e.appendRow([]any{
		dbName,
		tblName,
		partitionName,
		e.versionToTime(statsTbl.Version),
		statsTbl.ModifyCount,
		statsTbl.RealtimeCount,
		lastAnalyzeTime,
		lastStatsFullUpdateTime,
	})
```

Not sure if this is better or not. At least we don't need to use the index anymore.

Reply:
I reverted the changes of `show stats_meta` first, to keep this pr simple.